.github/workflows/iroha2-dev-pr-static.yml

```diff
@@ -28,13 +32,13 @@ jobs:
       - name: Format check
         run: cargo +nightly-2021-10-22 fmt --all -- --check
       - name: Static analysis without features
-        run: cargo lints clippy --workspace --benches --tests
+        run: cargo lints clippy --workspace --benches --tests --quiet
```

Comment:
why are we going quiet everywhere?

Reply:
Because this silences the messages we aren't interested in: like `Compiling XXX`, and passing tests.